app/src/main/res/layout/list_item.xml

```diff
@@ -25,58 +25,33 @@
             android:layout_marginLeft="10dp"
             android:text="Name"
             android:textColor="#000"
-            android:textSize="25sp"
+            android:textSize="18sp"
             android:textStyle="bold" />
 
-        <TextView
-            android:id="@+id/coursetext"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_below="@id/nametext"
-            android:layout_alignParentStart="true"
-            android:layout_marginStart="25dp"
-            android:layout_marginLeft="10dp"
-
-            android:text="Course Name"
-            android:textColor="#000"
-            android:textSize="20sp" />
-
-        <TextView
-            android:id="@+id/emailtext"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_below="@id/coursetext"
-            android:layout_alignParentStart="true"
-            android:layout_marginStart="15dp"
-            android:layout_marginLeft="10dp"
-
-            android:text="Email"
-            android:textColor="#000"
-            android:textSize="20sp" />
 
         <Button
             android:id="@+id/button"
-            android:layout_width="150dp"
-            android:layout_height="53dp"
-            android:layout_below="@+id/emailtext"
+            android:layout_width="148dp"
+            android:layout_height="82dp"
+            android:layout_below="@+id/nametext"
             android:layout_alignParentStart="true"
             android:layout_alignParentBottom="true"
-            android:layout_marginStart="15dp"
-            android:layout_marginTop="23dp"
-            android:layout_marginBottom="8dp"
+            android:layout_marginStart="39dp"
+            android:layout_marginTop="44dp"
+            android:layout_marginBottom="53dp"
             android:text="Button" />
```

Comment:
Please rename the Button text to its appropriate name even for placeholder purposes

Reply:
Created new XML file called item_list_pending with the appropriate names for widgets